app/controllers/api/teams.php

```diff
@@ -304,7 +306,9 @@
     ->label('sdk.response.model', Response::MODEL_MEMBERSHIP)
     ->label('abuse-limit', 10)
     ->param('teamId', '', new UID(), 'Team ID.')
-    ->param('email', '', new Email(), 'Email of the new team member.')
+    ->param('email', '', new Email(), 'Email of the new team member.', true)
+    ->param('userId', '', new UID(), 'User ID.', true)
```

Comment:
```suggestion
    ->param('userId', '', new UID(), 'Unique Id. Choose your own unique ID or generate a random ID with `ID.unique()`. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.', true)
```

Reply:
@gewenyu99 we don't allow using new ID here, it should be an existing User ID so that we can add the existing user to the team.